packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -513,9 +592,37 @@ export const prepareCosmosOrchestrationAccountKit = (
         },
 
         /** @type {HostOf<OrchestrationAccountI['transfer']>} */
-        transfer(amount, msg) {
-          console.log('transferSteps got', amount, msg);
-          return asVow(() => Fail`not yet implemented`);
+        transfer(amount, destination, opts) {
+          trace('transfer', amount, destination, opts);
+          return asVow(() => {
+            const { helper } = this.facets;
+            const token = helper.amountToCoin(amount);
+
+            const connectionInfoV = watch(
+              chainHub.getConnectionInfo(
+                this.state.chainAddress.chainId,
+                destination.chainId,
+              ),
+            );
+
+            // set a `timeoutTimestamp` if caller does not supply either `timeoutHeight` or `timeoutTimestamp`
+            // TODO #9324 what's a reasonable default? currently 5 minutes
+            const timeoutTimestampVowOrValue =
+              opts?.timeoutTimestamp ??
+              (opts?.timeoutHeight
+                ? 0n
+                : E(timestampHelper).getTimeoutTimestampNS());
+
+            // Resolves when host chain successfully submits, but not when
```

Comment:
the commit message made me think the feature was implemented. I see it just says it's adding the method, and I suppose there will be another `feat` commit that it works as expected.